napari/layers/image/_tests/test_big_image_timing.py

```diff
@@ -11,35 +11,30 @@
 data_dask_2D = da.random.random((100_000, 100_000))
 
 
-@pytest.mark.timeout(2)
 @pytest.mark.parametrize('data', [data_dask, data_zarr])
```

Comment:
The reason "timeout" was used here is that this test (and others like it) is trying to make sure that we don't instantiate the whole dask array, so the the expected "failure mode" of this test is that it takes too long - hence the timeout. Without the timeout things will just "take forever" or "crash". Are you sure this is the preferable option / is there nothing else in the ecosystem we can use here that is a bit safer?

Reply:
thanks for the clarification.  that seems like something we can do manually... lemme take a look

Reply:
i played with this for a while, but i don't think it's worth the time it's taking me.  I added back timeout just to these tests, and returned it to the dependencies